README.md

```diff
@@ -18,7 +18,7 @@ Install prerequisite RPMs if necessary:
 
 * modularity-testing-framework - Modularity test framework
 
-        $ sudo dnf copr enable phracek/Modularity-testing-framework
+        $ sudo dnf copr enable phracek/meta-test-family
         $ sudo dnf install modularity-testing-framework
```

Comment:
Since the COPR repo is being updated to reflect the renamed package, is there a reason _not_ to update the package to install to `meta-test-family` as well? (And possibly also the section heading a few lines above?)